docker/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
ENTRYPOINT script for all Dockerfile tests. Validates composer.json, installs dependencies, and runs our test suite. Finally echos PHP version for tracking purposes.